packages/core/src/interfaces/charts.ts

```diff
@@ -147,3 +147,29 @@ export interface DonutChartOptions extends PieChartOptions {
 		};
 	};
 }
+
+export interface MeterChartOptions extends BaseChartOptions {
+	meter?: {
+		barHeight?: number,
```

Comment:
Wouldn't this just be called `height`? So you'd access it as `chartOptions.meter.height`